webapp/src/components/__tests__/ProgressBar.test.tsx

```diff
@@ -0,0 +1,10 @@
+import { render, screen } from '@testing-library/react';
+import ProgressBar from '../ProgressBar';
+import React from 'react';
+
+describe('testing if progress bar inside Profile component is rendering', () => {
+  it('should render progress bar', () => {
+    render(<ProgressBar />);
+    expect(screen.getByText('Learning Progress')).toBeInTheDocument();
+  });
```

Comment:
Awesome job adding a test!

Reply:
Thank you Shea! It was a great team effort for our very first test.
We did encounter an error during the push and fixed it.
I'm going to document the problem and solution and post it in google drive documentation folder.

Reply:
Thank you, Shea!